README.md

```diff
@@ -408,5 +408,6 @@ fixing bugs:
  * johnsonlee
  * Philip Prindeville
  * Vis Virial
+ * Mike Bourgeous
```

Comment:
Thought I'd save you the trouble :).

If you think this is not a big enough bug fix, feel free to remove this.